cogs5e/funcs/scripting/evaluators.py

```diff
@@ -248,7 +253,7 @@ async def run_commits(self):
     def needs_char(self, *args, **kwargs):
         raise FunctionRequiresCharacter()  # no. bad.
 
-    def set(self, name, value):
+    def set(self, name, value):  # todo
```

Comment:
on the todo: `set()` is a function in python to create a set, but in Draconic 1.0 it was used to set variables (before `x=y` syntax, you had to do `set('x', y)`). Removing it would be a breaking change - this todo is here to remind me to have a conversation with community alias authors about that.

For the initial 2.0 release, `set()` will stay in (this todo can be safely ignored).

Speaking of which - @Croebh @thecoppersanctum @silverbass